src/sswsdk/dbconnector.py

```diff
@@ -9,6 +9,9 @@
 
 
 class SonicV1Connector(DBInterface):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+
```

Comment:
https://docs.python.org/2/library/functions.html#super
class C(B):
    def method(self, arg):
        super(C, self).method(arg) # I used to such call of the parent constructor. I'm not sure will super() work for python 2.